js/privatebin.js

```diff
@@ -1512,7 +1512,12 @@ jQuery.PrivateBin = (function($, RawDeflate) {
         me.getPasteKey = function()
         {
             if (symmetricKey === null) {
-                let newKey = window.location.hash.substring(1);
+                let pos = 1;
+                const pt = '#protected/';
```

Comment:
So this differenciates a burn after reading paste from normal pastes from now on?

If so, I would rather name it like:
```suggestion
                const prefix = '#onetime/';
```

or:
```suggestion
                const prefix = '#selfdestroy/';
```